### ISSUE TITLE
URL Drop window and system tray icon disappear after some time

### ISSUE THREAD

**Issue by redactedscribe** [erickutcher/httpdownloader#211]
It's unclear why, but after some time the app's system tray icon disappears, as does its URL drop window, meaning to access the program one has to open it via the Start Menu (single-instance mode is on). The program remains functional but I usually just exit the process and then start it again to remedy the situation, bringing back both features. Alternatively, it looks like I can also just go into the settings and uncheck, apply, and then re-apply both _Enable System Tray icon_ and _Enable URL drop window_ to get things back to normal.

Unfortunately, I don't know why this is happening. It could be related to waking from a sleep/hibernation state, though I think it also happens at random points during the day (I only notice when it comes to downloading something). It occurs at least once every two days. No other programs have been losing their icon presence in the system tray, so I'm currently under the impression that this is HTTP Downloader's issue.

Thanks.

HTTP Downloader 1.0.4.9
Windows 11 21H2

**Comment by erickutcher** [comment]
The only thing that I can think of that would cause them to disappear is if Explorer crashed. It could also be an issue with Windows 11. I don't have a copy of that system to test it out on unfortunately.

**Comment by redactedscribe** [comment]
I have noticed that my desktop has had its icons rearranged a number of times in the last month so I think you might be right that that Explorer is the underlying issue. It is odd however that 15 or so other system tray icons do not disappear. I'll have to keep an eye on it an report back if anything else comes to light.

**Comment by seesharpdeveloper** [comment]
For me it's the same. on windows11 it's disappeared without any notices, 
Also when a download is done and a notification pops up, when you click on the notification, it closes the app permanently.

**Comment by erickutcher** [comment]
I won't really be able to test this until I install Windows 11. Not sure when that will be.

As far as the notifications go, when you click on it, it should display the main window. If it closes it, then there's something else going on. The code for that shouldn't close anything or crash the program.

**Comment by seesharpdeveloper** [comment]
yes, it's weird, today I tested it on my other laptop with Windows 10, there is not such issue. 
Hope you find a chance/time to install win11
BTW thanks,
